src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -463,13 +513,33 @@ private void updateCitizens()
     }
 
     /**
+     * Takes WorkOrderView and compares their priorities, and spits out a nummber accordingly for use in the sortWorkOrders function
+     *  - It's here because Sonar doesn't like nested ternary operations. I'd say it's ugly but it works - (i.e. feel free to replace it if it annoys you)
+     * @param First
+     * @param Second
+     * @return
+     */
+    private int returnSecondOrFirst(WorkOrderView First, WorkOrderView Second)
+    {
+            return Second.getPriority() < First.getPriority() ? -1 : 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)